code/modules/mob/living/simple_animal/hostile/mining_mobs/goldgrub.dm

```diff
@@ -133,8 +133,11 @@
 		qdel(src)
 
 /mob/living/simple_animal/hostile/asteroid/goldgrub/bullet_act(obj/projectile/P)
-	visible_message(span_danger("The [P.name] is repelled by [name]'s girth!"))
-	return BULLET_ACT_BLOCK
+	if(istype(P, /obj/projectile/colossus))
```

Comment:
I think it should only block if the goldgrub is alive instead

Reply:
https://github.com/tgstation/tgstation/pull/72635/commits/78afdd3db7b10ad5af8e55fcd83bb68a5ddf680e